docs/framework/install/on-windows-10.md

```diff
@@ -4,22 +4,21 @@ description: Learn how to install the .NET Framework on Windows 10.
 author: rlander
 ms.author: mairaw
 keywords: .NET Framework, Install
-ms.date: 08/03/2017
+ms.date: 10/16/2017
 ms.topic: article
 ms.prod: .net-framework
 ---
-
 # Install the .NET Framework on Windows 10
 
 You may need the .NET Framework to run an app on Windows 10. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
 
-## .NET Framework 4.7
+## .NET Framework 4.7.1
 
-The .NET Framework 4.7 is included with the [Windows 10 Creators Update](https://www.microsoft.com/software-download/windows10).
+The .NET Framework 4.7.1 is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10).
 
-If you're on an earlier version of Windows 10, you need to upgrade to [Windows 10 Creators Update](https://www.microsoft.com/software-download/windows10) to get the .NET Framework 4.7.
+If you're on an earlier version of Windows 10, you need to upgrade to [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10) to get the .NET Framework 4.7.1.
```

Comment:
This isn't accurate; 4.7.1 is supported on Windows 10 Anniversary Update and Windows 10 Creators Update.